### DISCUSSION TITLE
Bulk Update if we have one Edit button on the top of the Table.

### DISCUSSION THREAD

**Discussion by sandeepbrightspeed** [KevinVandy/material-react-table#679] [General]
Just Want to know weather can we do Bulk update for multiple rows at one time using this library?
Suppose if we have one edit button on the top of Table and if we click on that edit button every row cells should be editable.
And if user edit those cells and user click on on more button that is update on top of the table again.
Then every changed cell value should be updated with updated value. We have one api for bulk update.

**Comment by SandeepAgrahari** [comment]
Can anyone pls help me?
